packages/flutter/lib/src/material/stepper.dart

```diff
@@ -124,6 +124,57 @@ class Step {
 /// to this widget based on some logic triggered by the three callbacks that it
 /// provides.
 ///
+/// {@tool sample --template=stateful_widget_scaffold_center}
+///
+///```dart
+///  int _index = 0;
+///  Widget build(BuildContext context) {
+///    return Container(
+///      height: 300,
+///      width: 300,
+///      child: Stepper(
+///        currentStep: _index,
+///        onStepCancel: () {
+///          if (_index <= 0 ) {
+///           return null;
+///          }
+///          setState(() {
+///            _index--;
+///          });
+///        },
+///        onStepContinue: () {
+///          if(_index >= 1) {
+///           return null;
+///          }
+///          setState(() {
+///            _index++;
+///          });
+///        },
+///        onStepTapped: (index) {
+///          setState(() {
+///            _index = index;
+///          });
+///        },
+///        steps: [
+///          Step(
+///            title: Text("Step 1 title"),
+///            content: Container(
+///              alignment: Alignment.centerLeft,
+///              child: Text("Content for Step 1")
+///            ),
+///          ),
+///          Step(
+///            title: Text("Step 2 title"),
+///            content: Text("Content for Step 2"),
+///          ),
+///        ],
+///      ),
+///    );
+///  }
```

Comment:
Everything is one space too far. It should be
```
/// Line of Code
```

instead of 
```
///  Line of Code
```

Reply:
Got it! Thanks :D